mat64/eigen.go

```diff
@@ -817,3 +820,49 @@ func (f EigenFactors) D() *Dense {
 	}
 	return dm
 }
+
+// MarshalBinary encodes the receiver into a binary form and returns the result.
+func (f EigenFactors) MarshalBinary() ([]byte, error) {
```

Comment:
@kortschak  extracting relevant data is one thing, re-synthesizing it (w/o too much of hysteresis) is another can of worms (especially when you don't have direct access to all data fields)